component-sets/default-components/components-definition.json

```diff
@@ -279,6 +280,19 @@
             "dataType": "inlineStyles",
             "featureFlag": "ContentStation-LocalStyleOverrides"
         },
+        {
+            "name": "chart-height",
+            "label": { "key": "CHART_HEIGHT__LABEL" },
+            "control": {
+                "type": "text",
+                "pattern": "/^(([+]{0,1})([0-9]|[0-9][.][0-9]{1,}|[1-9]{2}|[1-9]{2}[.][0-9]{1,}|[1-9][0-9]{0,2}|[1-4][0-9]{0,2}[.][0-9]{1,}|(500))|auto)$",
```

Comment:
A couple things I've noticed about this regex pattern:
1. Isn't the `500` case already included in the rest of the regex?
2. I also see that you've included a `/` at the beginning of the pattern, but are not closing with it. Is that on purpose?
3. Isn't the [+] case a bit redundant, since a positive number (without a `-` prefixed) automatically states that it is positive?
4. Why are decimal numbers above 49,99 not allowed but whole number up to 999 are fine?

Reply:
This regex was originally copied from the the `letter-spacing` and `line-height` patterns, but you are right, it can be simplified. I have updated all of them. This regex should do the trick : `^([0-9]{1,3}([.][0-9]{1,})?|auto)$`